packages/web/services/institutions.js

```diff
@@ -1,15 +1,21 @@
 import { apiGet, apiPost } from './api';
+import { HEADER as apiHeaderEnum } from '../utils/enums/api.enum';
 
 /**
  * Fetches institutions.
  *
+ * @param {object} options Optional params
```

Comment:
The default option param must be an empty object
```suggestion
 * @param {{}} options Optional params
```